pkg/client/mocks/BackupProvider.go

```diff
@@ -30,17 +30,24 @@ func (_m *BackupProvider) Abort(ctx context.Context, remoteName string, request
 }
 
 // Create provides a mock function with given fields: ctx, remoteName, request
-func (_m *BackupProvider) Create(ctx context.Context, remoteName string, request client.BackupCreate) error {
+func (_m *BackupProvider) Create(ctx context.Context, remoteName string, request client.BackupCreate) (string, error) {
```

Comment:
why did this happen in this commit?

Reply:
I updated golinstor, which changed the signature for that function to return the snapshot name. So I had to rerun the mock generation

Reply:
ah, that was it. thanks.